### ISSUE TITLE
Single HAT reaction overwrites input files

### ISSUE THREAD

**Issue by ehhartmann** [hits-mbm-dev/kimmdy#339]
To ensure we don't change the input files, all input files could be written to 0_setup and then discovered. Any objections?

**Comment by jmbuhr** [comment]
agreed. This also makes the output directory fully self-contained and reproducible for a run.
